runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/TypographyWidget.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+
+import { InfoOutlined } from '@mui/icons-material';
+import { Box, Typography, TypographyProps } from '@mui/material';
+import { WidgetProps } from '@rjsf/core';
+
+import If from 'src/components/utils/If';
+import { lightTheme } from 'src/theme/light';
+
+interface TypographyWidgetProps extends WidgetProps {
+    options: {
+        text: string;
+        variant: TypographyProps['variant'];
+        infoIcon: boolean;
+    }
+}
+
+const TypographyWidget: FC<TypographyWidgetProps> = ({ options }) => (
+    <Box display="flex" gap="5px" alignItems="center">
+        <If condition={options.infoIcon}>
+            <InfoOutlined
+                sx={{
+                    color: lightTheme.palette.infoIcon.default,
```

Comment:
that's not how you use `theme`. Now no matter if we switch between themes you always get the same result

In case you are not able to read the docs there are 2 options

# 1
![image](https://user-images.githubusercontent.com/44121335/204275718-7612018a-8ca5-459a-99a0-29e6b20912ed.png)

# 2
![image](https://user-images.githubusercontent.com/44121335/204275777-173ebc89-e4a8-4c8f-bd1e-9f71be1f22b8.png)